content/developer/reference/backend/testing.rst

```diff
@@ -819,35 +840,39 @@ driven explicitly by the user) it's more complicated when running
 "test" tours, or when running tours through the test suite. In that
 case there are two main tricks:
 
-- Have a step with a ``run() { debugger; }`` action.
+- Have a step property ``break: true,`` in debug mode (debug=True).
```

Comment:
I'd remove the "Have", it doesn't make much sense

Reply:
ok